openproblems/tasks/chromatin_potential/methods/linear_regression.py

```diff
@@ -86,7 +90,7 @@ def _atac_genes_score(adata, top_genes=500, threshold=1):
     adata.obsm["mode2"] = adata.obsm["mode2"][:, sel]
 
     # extend tss upstream and downstream
-    extend_tss = adata.var.iloc[:, -3:].apply(_chrom_limit, axis=1)
+    extend_tss = adata.var.loc[:, [3, 4, 5]].apply(_chrom_limit, axis=1)
```

Comment:
Better to use var names than positions, we can't assume these will always be the same.

Reply:
I add a set of var names for subsetting here.